sdk/healthcareapis/Microsoft.Azure.Management.HealthcareApis/src/Generated/Models/ServicesProperties.cs

```diff
@@ -45,19 +45,23 @@ public ServicesProperties()
         /// configuration of the service instance.</param>
         /// <param name="exportConfiguration">The settings for the export
         /// operation of the service instance.</param>
+        /// <param name="acrConfiguration">The azure container registry
+        /// settings used for convert data operation of the service
+        /// instance.</param>
         /// <param name="privateEndpointConnections">The list of private
         /// endpoint connections that are set up for this resource.</param>
         /// <param name="publicNetworkAccess">Control permission for data plane
         /// traffic coming from public networks while private endpoint is
         /// enabled. Possible values include: 'Enabled', 'Disabled'</param>
-        public ServicesProperties(string provisioningState = default(string), IList<ServiceAccessPolicyEntry> accessPolicies = default(IList<ServiceAccessPolicyEntry>), ServiceCosmosDbConfigurationInfo cosmosDbConfiguration = default(ServiceCosmosDbConfigurationInfo), ServiceAuthenticationConfigurationInfo authenticationConfiguration = default(ServiceAuthenticationConfigurationInfo), ServiceCorsConfigurationInfo corsConfiguration = default(ServiceCorsConfigurationInfo), ServiceExportConfigurationInfo exportConfiguration = default(ServiceExportConfigurationInfo), IList<PrivateEndpointConnection> privateEndpointConnections = default(IList<PrivateEndpointConnection>), string publicNetworkAccess = default(string))
+        public ServicesProperties(string provisioningState = default(string), IList<ServiceAccessPolicyEntry> accessPolicies = default(IList<ServiceAccessPolicyEntry>), ServiceCosmosDbConfigurationInfo cosmosDbConfiguration = default(ServiceCosmosDbConfigurationInfo), ServiceAuthenticationConfigurationInfo authenticationConfiguration = default(ServiceAuthenticationConfigurationInfo), ServiceCorsConfigurationInfo corsConfiguration = default(ServiceCorsConfigurationInfo), ServiceExportConfigurationInfo exportConfiguration = default(ServiceExportConfigurationInfo), ServiceAcrConfigurationInfo acrConfiguration = default(ServiceAcrConfigurationInfo), IList<PrivateEndpointConnection> privateEndpointConnections = default(IList<PrivateEndpointConnection>), string publicNetworkAccess = default(string))
```

Comment:
this will result in a breaking change, new parameters should be appended to the end